PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
So in FD (in Standalone mode) `BaseDir` and `AppDir` will still return the **exact** same paths, right?
If so, I understood your point. It was just the difference in understanding the definition of `BaseDir` - I knew it as `Standalone ? AppDir : UserAppDir` (it's also documented like this).
Your definition was `Standalone ? (PluginCore's directory) : UserAppDir`.

But since the cases you're describing has to deal with PluginCore's directory, and PluginCore.dll and FlashDevelop.exe are placed in the same folder in FD, I think it's okay.

Although I would prefer making a new property that returns `Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()))` with `GetAssemblyPath()` inlined in the property. This property could be needed in other scenarios too.

By the way, why is the last line of `GetAssemblyPath()` returning `Location` instead of `CodeBase`? And can't the whole thing be replaced with `new Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath`?